int64 rcRoll
int64 rcPitch
int64 rcYaw
int64 rcThrottle
int64 plutoIndex
int8 commandType
